a/apache-flink/apache-flink_master_rhel_8.3.sh

```diff
@@ -18,23 +18,68 @@ VERSION="master"
 
 # Install dependencies and tools.
 sudo yum update -y
-sudo yum install -y git wget java-1.8.0-openjdk-devel
+sudo yum install -y git wget java-1.8.0-openjdk-devel patch xz
 export JAVA_HOME=export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk
 
-#Install maven v3.2.5 which is recommended by Flink 1.8.x
-wget https://www-us.apache.org/dist/maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.tar.gz --no-check-certificate --quiet
-tar xzf apache-maven-3.2.5-bin.tar.gz
+# Install postgresql build dependencies
+sudo yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-ppc64le/pgdg-redhat-repo-latest.noarch.rpm
+sudo yum install -y postgresql10-libs
+sudo yum install -y postgresql10 postgresql10-server postgresql10-contrib
 
-export M2_HOME=`pwd`/apache-maven-3.2.5
-export PATH=`pwd`/apache-maven-3.2.5/bin:${PATH}
+# Install maven v3.6.3
+wget https://www-us.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz --no-check-certificate --quiet
+tar xzf apache-maven-3.6.3-bin.tar.gz
 
-#Clone and build source
+# Set ENV variables
+export M2_HOME=`pwd`/apache-maven-3.6.3
+export PATH=`pwd`/apache-maven-3.6.3/bin:${PATH}
+
+# Clone and build source
 git clone https://github.com/apache/flink.git
 cd flink
 git checkout $VERSION
 
 # Updated com.google.protobuf:protoc from 3.5.1-->3.7.0
 sed -i 's/3.5.1/3.7.0/g' flink-formats/flink-parquet/pom.xml
 
-#Compile and build package using threads
+# Compile and build package using threads
 mvn clean package -T 6 -DskipTests -Dfast
+
+# Resolve "No postgres binary for ppc64le" error
```

Comment:
can you add more comments to explain what you are doing below?

Reply:
Resolved. Updated the explanation.